.github/workflows/ccpp.yml

```diff
@@ -36,60 +25,39 @@ jobs:
 
     - name: Initialise
       run: mkdir build
-
+      
     - name: 'Setup vcpkg (Windows)'
```

Comment:
Food for thought: since Windows and Linux are separated into their own jobs with this PR, perhaps the `(Windows)` is no longer needed?